yt_dlp/extractor/zingmp3.py

```diff
@@ -212,11 +236,189 @@ class ZingMp3AlbumIE(ZingMp3BaseIE):
+        api = self.get_api_with_signature(name_api=self._SLUG_API[type_url], param={'id': type_url})
+        return self._process_data(self._download_json(api, type_url)['data'], type_url, type_url)
+
+    def _process_data(self, data, chart_id, type_url):
+        if type_url == 'zing-chart':
+            items = traverse_obj(data, ('RTChart', 'items'), default=[])
+        else:
+            items = data.get('items')
+        return self.playlist_result(self._entries(items), type_url, type_url)
+
+
+class ZingMp3WeekChartIE(ZingMp3BaseIE):
+    _VALID_URL = r'https?://(?:mp3\.zing|zingmp3)\.vn/(?P<type>zing-chart-tuan)/[^/?#]+/(?P<id>\w+)'
+    IE_NAME = 'zingmp3:week-chart'
+    _TESTS = [{
+        'url': 'https://zingmp3.vn/zing-chart-tuan/Bai-hat-Viet-Nam/IWZ9Z08I.html',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'IWZ9Z08I',
+            'title': 'zing-chart-vn',
+        },
+        'playlist_mincount': 10,
+    }, {
+        'url': 'https://zingmp3.vn/zing-chart-tuan/Bai-hat-US-UK/IWZ9Z0BW.html',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'IWZ9Z0BW',
+            'title': 'zing-chart-us',
+        },
+        'playlist_mincount': 10,
+    }, {
+        'url': 'https://zingmp3.vn/zing-chart-tuan/Bai-hat-KPop/IWZ9Z0BO.html',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'IWZ9Z0BO',
+            'title': 'zing-chart-korea',
+        },
+        'playlist_mincount': 10,
+    }]
+
+    def _process_data(self, data, chart_id, type_url):
+        return self.playlist_result(
+            self._entries(data['items']), chart_id, f'zing-chart-{data.get("country", "")}')
+
+
+class ZingMp3ChartMusicVideoIE(ZingMp3BaseIE):
+    _VALID_URL = r'https?://(?:mp3\.zing|zingmp3)\.vn/(?P<type>the-loai-video)/.+?/(?P<id>[^\.]+)'
+    IE_NAME = 'zingmp3:chart-music-video'
+    _TESTS = [{
+        'url': 'https://zingmp3.vn/the-loai-video/Viet-Nam/IWZ9Z08I.html',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'IWZ9Z08I',
+            'title': 'zing-chart-video',
+            'description': 'zing-chart-video',
+        },
+        'playlist_mincount': 400,
+    }]
+
+    def _fetch_page(self, song_id, type_url, page):
+        page += 1
+        api = self.get_api_with_signature(name_api=self._SLUG_API[type_url], param={
+            'id': song_id,
+            'type': 'genre',
+            'page': page,
+            'count': self._PER_PAGE
+        })
+        data = self._download_json(api, song_id)['data']
+        items = data.get('items')
+        if items:
+            return self._entries(items)
+
+    def _real_extract(self, url):
+        song_id, type_url = self._match_valid_url(url).group('id', 'type')
+        entries = OnDemandPagedList(functools.partial(self._fetch_page, song_id, type_url), self._PER_PAGE)
+        return self.playlist_result(entries, song_id, 'zing-chart-video', 'zing-chart-video')
```

Comment:
as I asked in other PRs, why set a title that is useless?

Reply:
Fixed.